pycon/tests/test_export.py

```diff
@@ -0,0 +1,128 @@
+from __future__ import unicode_literals
+
+from httplib import OK
+from zipfile import ZipFile
+from StringIO import StringIO
+from datetime import timedelta, date
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from django.utils.timezone import now
+from pycon.sponsorship.models import SponsorLevel, Sponsor, Benefit
+from pycon.tests.factories import PyConTutorialProposalFactory
+from symposion.conference.models import Conference, current_conference, Section
+from symposion.proposals.models import ProposalKind
+from symposion.schedule.models import Schedule, Slot, Day, SlotKind
+from symposion.schedule.tests.factories import PresentationFactory
+
+
+class ProgramExportTest(TestCase):
+    fixtures = [
+        'fixtures/conference.json',
+        'fixtures/proposal_base.json'
+    ]
+
+    def setUp(self):
+        self.url = reverse('program_export')
+        Conference.objects.get_or_create(id=settings.CONFERENCE_ID)
+        conference = current_conference()
+        self.sponsor_level = SponsorLevel.objects.create(
```

Comment:
There are existing factories for several of these things, in case you don't want to have to define unneeded data yourself...